x/slashing/errors.go

```diff
@@ -12,41 +12,16 @@ const (
 	// Default slashing codespace
 	DefaultCodespace sdk.CodespaceType = 10
 
-	// Invalid validator
-	CodeInvalidValidator CodeType = 201
-	// Validator jailed
-	CodeValidatorJailed CodeType = 202
+	CodeInvalidValidator CodeType = 101
+	CodeValidatorJailed  CodeType = 102
 )
 
 func ErrNoValidatorForAddress(codespace sdk.CodespaceType) sdk.Error {
-	return newError(codespace, CodeInvalidValidator, "that address is not associated with any known validator")
+	return sdk.NewError(codespace, CodeInvalidValidator, "That address is not associated with any known validator")
```

Comment:
I think we wanted lowercase error strings, nbd though

Reply:
addressed all in `x/slashing`